Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -459,10 +459,10 @@ private static double GetAspectRatio(ScheduleDirect.ImageData i)
             }
 
             StringBuilder str = new StringBuilder("[", 1 + (programIds.Count * 13));
-            foreach (ReadOnlySpan<char> i in programIds)
+            foreach (string i in programIds)
```

Comment:
I don't know which is better, but I know this was added on purpose

Reply:
👀 unintended change, I put that there to have Rider stop complaining and accidentally commited it